<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>codeeditor</class>
 <widget class="QMainWindow" name="codeeditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>codeeditor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>427</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionFind_and_Replace"/>
   </widget>
   <widget class="QMenu" name="menuTemplates">
    <property name="title">
     <string>Templates</string>
    </property>
    <addaction name="actionIf"/>
    <addaction name="actionIf_Else"/>
    <addaction name="actionFor"/>
    <addaction name="actionWhile"/>
    <addaction name="actionDo_While"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuSyntax_Validation">
    <property name="title">
     <string>Syntax Validation</string>
    </property>
    <addaction name="actionModify_Type_Declarations"/>
    <addaction name="actionModify_Active_Syntax_Rules"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTemplates"/>
   <addaction name="menuSyntax_Validation"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionFind_and_Replace">
   <property name="text">
    <string>Find and Replace</string>
   </property>
  </action>
  <action name="actionIf">
   <property name="text">
    <string>If () {}</string>
   </property>
  </action>
  <action name="actionIf_Else">
   <property name="text">
    <string>If (){} Else {}</string>
   </property>
  </action>
  <action name="actionFor">
   <property name="text">
    <string>For (){}</string>
   </property>
  </action>
  <action name="actionWhile">
   <property name="text">
    <string>While (){}</string>
   </property>
  </action>
  <action name="actionDo_While">
   <property name="text">
    <string>Do{} While ()</string>
   </property>
  </action>
  <action name="actionDeclaration_Wizard">
   <property name="text">
    <string>Declaration Wizard</string>
   </property>
  </action>
  <action name="actionCustom">
   <property name="text">
    <string>Custom</string>
   </property>
  </action>
  <action name="actionModify_Type_Declarations">
   <property name="text">
    <string>Modify Type Declarations</string>
   </property>
  </action>
  <action name="actionModify_Active_Syntax_Rules">
   <property name="text">
    <string>Modify Active Syntax Rules</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
